contrib/backporting/submit-backport

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Copyright 2020 Authors of Cilium
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DIR=$(dirname $(readlink -ne $BASH_SOURCE))
+source $DIR/../release/lib/common.sh
+source $DIR/common.sh
+
+BRANCH="${1:-}"
+if [ "$BRANCH" = "" ]; then
+    BRANCH=$(git symbolic-ref --short HEAD | sed 's/.*\(v[0-9]\.[0-9]\).*/\1/')
+fi
+BRANCH=$(echo "$BRANCH" | sed 's/^v//')
+
+SUMMARY=${2:-}
+if [ "$SUMMARY" = "" ]; then
+    SUMMARY="$(date --rfc-3339=date).txt"
+fi
+
+if ! git branch -a | grep -q "origin/v$BRANCH$" || [ ! -e $SUMMARY ]; then
+    echo "usage: $0 [branch version] [pr-summary]" 1>&2
+    echo 1>&2
+    echo "Ensure 'branch version' is available in 'origin' and the summary file exists" 1>&2
+    exit 1
+fi
+
+if ! gh help | grep -q "pull-request"; then
+    echo "This tool relies on 'gh' from https://github.com/node-gh/gh ." 1>&2
+    echo "Please install this tool first."
+    exit 1
+fi
+
+echo -e "Sending PR for branch v$BRANCH:\n" 1>&2
+cat $SUMMARY 1>&2
+echo -e "\nSending pull request..." 2>&1
+git push origin
+gh pull-request -b "v$BRANCH" -l kind/backports,backport/$BRANCH,requires-janitor-review -F $SUMMARY
```

Comment:
Is `gh` an alias or a tool that should be installed?

Reply:
It's a tool, already documented in backports.rst: https://www.npmjs.com/package/gh

Reply:
It's the tool that's already described earlier in the docs:

http://docs.cilium.io/en/latest/contributing/release/backports/#one-time-setup